testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestExternalHiveTable.java

```diff
@@ -149,6 +149,38 @@ public void testCreateExternalTableWithInaccessibleSchemaLocation()
         String table = "test_create_external";
         String tableLocation = "/tmp/" + table;
         query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema, table, tableLocation));
+
+        String otherSchema = "schema_with_unknown_location";
+        query(format("CREATE SCHEMA %s.%s", HIVE_CATALOG_WITH_EXTERNAL_WRITES, otherSchema));
+
+        String otherTable = "table_with_unknown_schema_location";
+        String otherTableLocation = "/tmp/" + otherTable;
+        query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, otherSchema, otherTable, otherTableLocation));
+    }
+
+    @Test(groups = {HIVE_WITH_EXTERNAL_WRITES, PROFILE_SPECIFIC_TESTS})
+    public void testCreateExternalTableWithExistingPath()
+    {
+        String schema = "schema_with_file_as_location";
+
+        String table = "table_with_file_as_location";
+        String tableLocation = "/tmp/" + table;
+        hdfsClient.saveFile(tableLocation, "This is a file");
+        assertThat(() -> query(
+                format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema, table, tableLocation)))
```

Comment:
I think we'd want to use CT, and not CTAS here right? the error message should say that the path exists but not a directory.

Reply:
I think we should have both here. A CT and a CTAS to verify that the CTAS behaviour matches the CT behaviour now.